### ISSUE TITLE
Installation from the PyPI with Python 3.6+ failed

### ISSUE THREAD

**Issue by ptynecki** [sacdallago/bio_embeddings#52]
Hey,

I discovered that _bio-embeddings_ installation from the PyPI with Python 3.6+ doesn't work anymore.
But it's working fine directly from the GitHub repository.

```bash
pip install bio-embeddings[all]

ERROR: Could not find a version that satisfies the requirement bio-embeddings[all] (from versions: none)
ERROR: No matching distribution found for bio-embeddings[all]
```

or 

```bash
ERROR: Could not find a version that satisfies the requirement bio-embeddings==0.1.3 (from versions: none)
```

I tried with Python 3.6.8 on Mac OX and with Python 3.6.9 on Ubuntu. The same raised exception.
I think it is related with the `setup.py` configuration:

```python
'python_requires': '>=3.7,<4.0'
```

Could you tell why Python 3.6 is excluded? Thanks!

**Comment by konstin** [comment]
Hi,

in previous releases, including the one that is on pypi, we only supported python 3.7+. We've only recently extended this to python 3.6. Unfortunately this means that `pip install bio-embeddings` won't work until we release v0.1.4 (which hopefully won't take too long).

**Comment by sacdallago** [comment]
Hey @ptynecki ,

another note: you install the pipeline like `pip install bio-embeddings[all]`, by this I guess you are reading the develop README. To install for the previous release it's sufficient to run `pip install bio-embeddings`, we had no extras at the time.

The version currently released on pypi is `0.1.3`, and while not much of the way we process pipeline runs has changed for the user since then, we have made quite a significant amount of changes to the APIs (preserving, mostly, backwards compatibility).

If you are interested in knowing what the pipeline looked like in `0.1.3` and what was availbe then, just checkout [master](https://github.com/sacdallago/bio_embeddings/tree/master).

At the time, we didn't support python `3.6` (for no major reason), but since `0.1.4` we want to be backwards compatible also with python `3.6`, especially for the colab integration ;)

If you have any other questions, feel free to ask.
